python/core/utilities-i/schedule-events-with-sched.md

```diff
@@ -56,3 +56,45 @@ Then we run our scheduler:
 s.run()
 ```
 `first` is printed after 2 seconds while `second` is printed after 3 seconds.
+
+---
+## Practice
+
+Create a new schedule and set it to enter second with priority = 1 and delay = 4.
+
+```python
+
+import sched
+import time
+
+start = 2018-03-23 21:55:19.629694
```

Comment:
wait is this... how this actually works? without quotes and everything? I have never seen dates written like this, can you find an example (or fix it)